docs/releases/2.1.txt

```diff
@@ -423,6 +428,9 @@ Miscellaneous
   accept ``obj`` as the second positional argument will be removed in Django
   3.0.
 
+* ``RawQuerySet`` now implements internal results caching, that can be avoided by
+  using ``.iterator()`` instead.
```

Comment:
This is in the wrong section. (It's in _Features deprecated... > Misc_ it need to be in _BC > Misc_, which is just above.)

I'd rephrase slightly. Something like: 

```
``django.db.models.query.RawQuerySet`` now implements internal results caching. You may use ``RawQuerySet.iterator()`` to bypass the results cache.
```